Admin/Admin.php

```diff
@@ -2030,12 +2052,15 @@ public function getFilterFieldDescriptions()
     /**
      * {@inheritdoc}
      */
-    public function addChild(AdminInterface $child)
+    public function addChild(AdminInterface $child, $field = null)
     {
         $this->children[$child->getCode()] = $child;
 
         $child->setBaseCodeRoute($this->getCode().'|'.$child->getCode());
         $child->setParent($this);
```

Comment:
@robhunt3r @core23 @ju1ius : I have a problem with the line above : if you call addChild on the `Comment` admin from 2 different admin, the last one will be the parent, and the first will not. Isn't that an issue ?